CONTRIBUTING.md

```diff
@@ -301,37 +243,83 @@ This is optional but encouraged.
 Good commit messages explain what changed AND why you changed it.
 See [I've written a clear changelist description](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-written-a-clear-changelist-description).
 
-## Contributor License Agreement
 
-You will be asked to sign our Contributor License Agreement when you open a Pull Request. We are not
-asking you to assign copyright to us but to give us the right to distribute
-your code without restriction. We ask this of all contributors to
-assure our users of the origin and continuing existence of the code. You only
-need to sign the CLA once.
+## Build Camunda from source
 
-## Licenses
+Camunda is a multi-module Maven project. To **quickly** build all components,
+run the command: `mvn clean install -Dquickly` in the root folder.
 
-Zeebe, Operate, and Tasklist source files are made available under the
-[Camunda License Version 1.0](/licenses/CAMUNDA-LICENSE-1.0.txt) except for the parts listed
-below, which are made available under the [Apache License, Version
-2.0](/licenses/APACHE-2.0.txt).  See individual source files for details.
+> [!NOTE]
+> All Java modules in Zeebe are built and tested with JDK 21. Most modules use language level
+> 21, exceptions are: zeebe-bpmn-model, zeebe-client-java, zeebe-gateway-protocol,
+> zeebe-gateway-protocol-impl, zeebe-protocol and zeebe-protocol-jackson which use language level 8
+>
+> The Go client and zbctl are built and tested with Go 1.22
+>
+> The Java and the Go modules are built and tested with Docker 20.10.5 [with IPv6 support](https://docs.docker.com/config/daemon/ipv6/).
+>
+> Macs using arm64 (Apple Silicon) must install Rosetta as some dependencies do not have native arm binaries.
 
-Available under the [Apache License, Version 2.0](/licenses/APACHE-2.0.txt):
-- Java Client ([clients/java](/clients/java))
-- Go Client ([clients/go](/clients/go))
-- Exporter API ([exporter-api](/exporter-api))
-- Protocol ([protocol](/protocol))
-- Gateway Protocol Implementation ([gateway-protocol-impl](/gateway-protocol-impl))
-- BPMN Model API ([bpmn-model](/bpmn-model))
+For contributions to Camunda, building quickly is typically sufficient.
+However, users of Camunda are recommended to build the full distribution.
 
-If you would like to contribute something, or simply want to hack on the code
-this document should help you get started.
+To fully build the Zeebe distribution, run the command: `mvn clean install -DskipTests` in the root folder.
+This is slightly slower than building quickly but ensures the distribution is assembled completely.
+The resulting Zeebe distribution can be found in the folder `dist/target`, i.e.
```

Comment:
🔧 This should probably be adjusted for Camunda's mono repo

Reply:
👍 
@xomiamoore maybe you go through the build process yourself, and adjust this accordingly?

Reply:
@MaxTru @korthout, Are the steps the same as these, or different? I need a bit more context to get started here.